### PR TITLE
LOOP-1492: Add prescription section

### PR DESCRIPTION
**Pull request by rickpasetto**
Also add some space above top section

### CODE REVIEW

File: LoopKit/Prescription.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+//  Prescription.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 7/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+public protocol Prescription {
+    var datePrescribed: Date { get } // Date prescription was prescribed
+    var providerName: String { get } // Name of clinician prescribing
```
Comment (mpangburn) [inline LoopKit/Prescription.swift]: [nit] Want to make these `///` doc comments above the requirements?

File: LoopKitUI/Views/Settings Editors/TherapySettingsView.swift [hunk 2]
```diff
@@ -120,6 +123,20 @@ extension TherapySettingsView {
...
+    private var prescriptionSection: some View {
+        SectionWithEdit(isEditing: .constant(false),
+                        addExtraSpaceAboveSection: true,
+                        title: LocalizedString("Prescription", comment: "title for prescription section"),
+                        descriptiveText: prescriptionDescriptiveText,
+                        destination: EmptyView(), content: { EmptyView() })
+    }
+    
+    private var prescriptionDescriptiveText: String {
+        String(format: LocalizedString("Submitted by %@, %@", comment: "Format for prescription descriptive text (1: providerName, 2: datePrescribed)"),
```
Comment (mpangburn) [inline LoopKitUI/Views/Settings Editors/TherapySettingsView.swift]: [nit] You might consider numbering the format string arguments (i.e. `%1$@` and `%2$@`) to facilitate reordering for translators

File: LoopKit/Prescription.swift [hunk 3]
```diff
@@ -0,0 +1,12 @@
+//
+//  Prescription.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 7/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+public protocol Prescription {
```
Comment (mpangburn) [inline LoopKit/Prescription.swift:9]: Will a type conforming to Prescription (e.g. Tidepool's impl) do something other than hold these two properties? If not, consider making this a struct

**Review by mpangburn — APPROVED** [review]
👍

**Review by novalegra — APPROVED** [review]
👍

File: LoopKit/Prescription.swift (hunk 3 above)
Comment (rickpasetto) [reply LoopKit/Prescription.swift:9]: I don't know the answer to this, yet. I'm not sure anyone does...which is specifically why I made it a `protocol`.

**Pull request merged**